src/save.ts

```diff
@@ -28,14 +24,10 @@ async function uploadBuildLog(): Promise<void> {
   const artifactClient = artifact.create()
   const artifactName = 'buildcache_log'
 
-  const installDir = await getInstallDir()
-  const cacheDir = getEnvVar(
-    'BUILDCACHE_DIR',
-    path.join(installDir, '.buildcache')
-  )
-  const logFile = getEnvVar(
-    'BUILDCACHE_LOG_FILE',
-    path.join(cacheDir, 'buildcache.log')
+  const cacheDir = await getCacheDir()
+  const logFile = path.resolve(
```

Comment:
same query as above on this one, just noting second ocurrence

Reply:
Same answer :)